sdk/cosmosdb/cosmos/src/request/ErrorResponse.ts

```diff
@@ -55,7 +55,7 @@ export interface GroupByAliasToAggregateType {
   [key: string]: AggregateType;
 }
 
-export interface ErrorResponse extends Error {
```

Comment:
This is a breaking change. Has to be reverted (or avoid breaking change otherwise)